src/styl/_helpers/_colors.styl

```diff
@@ -24,5 +45,8 @@ for key, value in $helpers-colors
     .bg-{$name}
         background-color $var
 
+    .fill-{$name}
```

Comment:
Maybe we should put it in its own file to be able to not load it in AMP css, if not used.